vowpalwabbit/io_to_queue.h

```diff
@@ -0,0 +1,76 @@
+#ifndef _IO_QUEUE_
+#define _IO_QUEUE_
+
+#include <thread>
+#include <future>
+#include <mutex>  
+#include <chrono>   
+
+#include "io_item.h"
+
+//Adds a line of input to the input queue
+inline bool add_to_queue(vw& all, char *& line){
```

Comment:
nit: bracket should be on the next line. You can use `clang-format` and the `.clang-format` file in the root of the repo to quickly resolve these little formatting things.